source/quick-start.txt

```diff
@@ -37,20 +37,25 @@ to check your version of Node and npm, as well as installation
 instructions, see `downloading and installing Node.js and npm 
 <https://docs.npmjs.com/downloading-and-installing-node-js-and-npm>`_.
 
+Select the :guilabel:`Clone the Project` tab if you want to build your
+project from a premade dependency file or the :guilabel:`Create the Project`
+tab if you want to create the project dependency file yourself.
+
 .. tabs::
 
    .. tab:: Clone the Project
       :tabid: clone the project
 
-      First, install Git using `Git's Getting Started Guide
+      First, ensure you have ``git`` installed in your development
+      environment or otherwise install Git using `Git's Getting Started Guide
       <https://git-scm.com/book/en/v2/Getting-Started-Installing-Git>`__.
 
-      Then, in your shell, clone the `js-starter repository
+      Then, from your shell, clone the `js-starter repository
       <https://github.com/mongodb-university/js-starter>`__:
 
       .. code-block:: bash
 
-         git clone https://github.com/mongodb-university/js-starter.git
+         git clone --branch v{+version+} https://github.com/mongodb-university/js-starter.git
```

Comment:
Praise: Awesome job versioning the starter project, really like how version-sensitive the docs are becoming.